helmfile/charts/networkpolicy-service/templates/certmanager-np.yaml

```
@@ -0,0 +1,129 @@
+{{ if .Values.certManager.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-deny-all
+  namespace: cert-manager
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+    - Egress
+
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-np
```

Since this is of the kind networkpolicy, do we need to specify in the name that it's a np?
```suggestion
  name: cert-manager
```

---

okay !! i can change it